ViAn/GUI/projectwidget.cpp

```diff
@@ -616,6 +632,7 @@ void ProjectWidget::open_project(QString project_path) {
 void ProjectWidget::close_project() {
     // TODO Check for unsaved changes before closing
     if (m_proj == nullptr) return;
+    parentWidget()->parentWidget()->setWindowTitle(QString::fromStdString(""));
```

Comment:
Create function "set_main_window_name" or similar to make it more clear of what happens here and in ProjectWidget::remove_project.